3520/0/about_java_interop.rb

```diff
@@ -0,0 +1,138 @@
+# rubocop:disable all
+require File.expand_path(File.dirname(__FILE__) + '/neo')
+
+include Java
+
+# Concepts
+# * Pull in a java class
+# * calling a method, Camel vs snake
+# * Resolving module/class name conflicts
+# * Showing what gets returned
+# * Ruby Strings  VS Java Strings
+# * Calling custom java class
+# * Calling Ruby from java???
+
+class AboutJavaInterop < Neo::Koan
+  def test_using_a_java_library_class
+    java_array = java.util.ArrayList.new
+    assert_equal __, java_array.class
```

Comment:
Why you didn't do that file?

Reply:
I use ruby 2.6 and that file are for using jruby. If I must install it and done this task - I'll do it.

Reply:
Ok. You can miss it